projects/novo-elements/src/elements/tooltip/Tooltip.directive.ts

```diff
@@ -71,6 +73,14 @@ export class TooltipDirective implements OnDestroy, OnInit {
     }
   }
 
+  @HostListener('click')
+  onclick(): void {
+    if (this.overlayRef && !this.always && this.closeOnClick) {
```

Comment:
Is it worth adding a unit test for this?

Reply:
there werent any unit tests for the other tooltip event listeners. should i still go ahead and add one for this?

Reply:
If it's fast, you could add it, if it ends up taking lots of setup, it's fine to skip.

Reply:
added unit test